samples/client/petstore/dart/flutter_petstore/openapi/lib/model/api_response.dart

```diff
@@ -2,11 +2,11 @@ part of openapi.api;
 
 class ApiResponse {
   
-  int code = null;
+    int code = null;
```

Comment:
@nickmeinhold looks like there are 2 additional space inserted. Is that intentional?

Reply:
No, not intentional - I didn't notice that sorry.  At first glance I don't understand where the extra spaces have come from as the only change I made to the template was inside the fromJson part. Any thoughts?  I will have a look tonight and see if I can understand it.

Reply:
Yeah I really don't get this - these lines are presumably generated by line4 of `class.mustache` and I haven't changed it or anything else nearby.  I'm still learning mustache, there must be something about how the indentation works that I'm missing.  I will read up more on mustache when I get some time, probably won't be today though.

Reply:
Don't worry. I'll take care of the formating with another PR